README.md

```diff
@@ -115,14 +104,31 @@ response.fail(String message)
 response.say("OK").send()
 ```
 
+## session
+```javascript
+// check if you can use session (read or write)
+Boolean session.isAvailable()
```

Comment:
It's a little strange to ask the session whether it's available. Why wouldn't this be on `request.hasSession()`? And then I think we don't need to change the first `function` because `request.session` would be a thing?